### ISSUE TITLE
v6.0.0 resolved path issue

### ISSUE THREAD

**Issue by aannoune** [dwyl/aws-sdk-mock#403]
Hello, thanks for the work you are doing. 
I'm having an issue with version 6.0.0
the main and types in you package.json is referring to the dist directory witch is dot delivered in the package:
package.json:
```json 
{
  "name": "aws-sdk-mock",
  "version": "6.0.0",
  "description": "Functions to mock the JavaScript aws-sdk",
  "type": "module",
  "main": "src/index.js",
  "types": "src/index.d.ts",
  "scripts": {
    "lint": "tsc",
    "nocov": "ts-mocha test/**/*.spec.ts",
    "test": "nyc ts-mocha test/**/*.spec.ts && tsd",
    "coverage": "nyc --report html ts-mocha test/**/*.spec.ts && open coverage/index.html",
    "build": "tsup src/index.ts --format cjs, esm, --dts"
  }
```
package content : 

```bash
aws-sdk-mock/
├── CONTRIBUTING.md
├── LICENSE
├── README.md
├── node_modules
│   ├── @sinonjs
│   └── sinon
├── package.json
├── src
│   ├── index.ts
│   └── types.ts
├── test
│   └── index.spec.ts
└── tsconfig.json
```
this result in a 
resoled path issue.

**Comment by LuchoTurtle** [comment]
Hey @aannounex, thanks for opening an issue 😎

I wager that fixing this problem might be as simple as just using the relative path, like `index.d.ts` instead of using `src`.

I'm travelling right now but I'll take a look at it as soon as I can. ☺️

**Comment by LuchoTurtle** [comment]
Following the advice from https://mikeesto.com/posts/publishing-npm-package/ and https://cameronnokes.com/blog/the-30-second-guide-to-publishing-a-typescript-package-to-npm/, I think I've found the problem. I've tested using `npm pack` to create a tarball to see how the files are bundled before publishing to `npm` and I've already committed some changes on a separate branch.

My internet here is funky and really slow, so it's hard to push branch.
